ibm/service/catalogmanagement/resource_ibm_cm_account.go

```diff
@@ -0,0 +1,424 @@
+													Computed:    true,
+													Optional:    true,
+													Description: "List of filter terms for the category.",
+													Elem: &schema.Schema{
+														Type: schema.TypeString,
+													},
+												},
+											},
+										},
+									},
+								},
+							},
+						},
+						"id_filters": &schema.Schema{
+							Type:        schema.TypeList,
+							Optional:    true,
+							Computed:    true,
+							Description: "Filter on offering ID's. There is an include filter and an exclule filter. Both can be set.",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"include": &schema.Schema{
+										Type:        schema.TypeList,
+										Optional:    true,
+										Computed:    true,
+										Description: "Offering filter terms.",
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"filter_terms": &schema.Schema{
+													Type:        schema.TypeList,
+													Optional:    true,
+													Computed:    true,
+													Description: "List of values to match against. If include is true, then if the offering has one of the values then the offering is included. If include is false, then if the offering has one of the values then the offering is excluded.",
+													Elem:        &schema.Schema{Type: schema.TypeString},
+												},
+											},
+										},
+									},
+									"exclude": &schema.Schema{
+										Type:        schema.TypeList,
+										Optional:    true,
+										Computed:    true,
+										Description: "Offering filter terms.",
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"filter_terms": &schema.Schema{
+													Type:        schema.TypeList,
+													Optional:    true,
+													Computed:    true,
+													Description: "List of values to match against. If include is true, then if the offering has one of the values then the offering is included. If include is false, then if the offering has one of the values then the offering is excluded.",
+													Elem:        &schema.Schema{Type: schema.TypeString},
+												},
+											},
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			"region_filter": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Region filter string.",
+			},
+		},
+	}
+}
+
+func resourceIBMCmAccountCreate(context context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	catalogManagementClient, err := meta.(conns.ClientSession).CatalogManagementV1()
+	if err != nil {
+		tfErr := flex.DiscriminatedTerraformErrorf(err, err.Error(), "ibm_cm_account", "read", "initialize-client")
+		log.Printf("[DEBUG]\n%s", tfErr.GetDebugMessage())
+		return tfErr.GetDiag()
+	}
+
+	getCatalogAccountOptions := &catalogmanagementv1.GetCatalogAccountOptions{}
```

Comment:
Why can't we handle the create as update. We are forcing the user's to do 2 steps for just applying optional filters.

Reply:
Not sure I follow, the create does call `return resourceIBMCmAccountUpdate(context, d, meta)`

Reply:
create calls resourceIBMCmAccountUpdate that should be fine